hw/top_earlgrey/util/opentitan_earlgrey_flash_size_reduce.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Script used to reduce the size of the embedded flash for smaller FPGA devices.
+
+This script modifies two source files to reduce the size of the embedded flash and then regenerates
+all auto-generated files. The flash size is reduced in a way that does not impact the register
+interface of the flash controller and is thus more or less transparent to software.
+"""
+
+import logging as log
+import subprocess
+import sys
+import os
+import re
+
+# Display INFO log messages and up.
+log.basicConfig(level=log.INFO, format="%(levelname)s: %(message)s")
+
+
+def main():
+
+    # Get path to top-level directory
+    top_path = os.path.normpath(os.path.join(os.path.dirname(__file__), "../../.."))
+    if (top_path == "."):
+        top_path = ""
+    else:
+        top_path += "/"
```

Comment:
Remove the if/else top_path part, and ...

Reply:
Done